### DISCUSSION TITLE
Wyliczenie użycia Logic Actions

### DISCUSSION THREAD

**Discussion by dzejzipl** [AzureCommunityPL/group#378] [Pomoc] [answered]
Hej, potrzebuję pomocy z wyliczeniem kosztów Logic Actions.
Przyklad:
1) Sprawdzam RSS bloga co 15 minut - 4 razy w ciągu godziny, na dzień daje nam to 96 razy. Ale z tego co czytam, opłata jest tylko wykonana jeżeli zostanie wykonana jakaś akcja. Np. pojawi się jakis wpis, zgadza sie?
2) Czyli na miesiąc będzie to 2976 wykonań - według cennika będzie to niecałe 0.065472 euro? 6 centów?!
Czy ja coś źle licze? 
Plan jest taki, że ma być sprawdzany RSS bloga co 15 minut i wrzucana akcja np. do sharepointa. Albo cokolwiek innego.

**Comment by kamil-mrzyglod** [comment, marked as answer]
Generalnie przy Consumption masz pierwsze 4000 akcji za free więc istnieje szansa, że na to się zmieścisz. Natomiast masz dość jasno w dokumentacji napisane:

> Except for the initial number of free built-in operation executions, per Azure subscription, that a workflow can run, the Consumption model meters and bills an operation based on each execution, whether or not the overall workflow successfully runs, finishes, or is even instantiated.

[Link](https://docs.microsoft.com/en-us/azure/logic-apps/logic-apps-pricing)

Więc pytanie do którego fragmentu się odnosisz pisząc o braku opłaty, jeśli nie zostanie wykonana akcja.

**Reply by dzejzipl** [reply]
Rozumiem to tak, że po upłynięciu darmowych akcji każdorazowe wykonanie co 15 minut będzie platne, niezależnie od tego czy zostanie do końca wykonane. Jeżeli dobrze liczę - 0.000022 euro za jedną akcję. Czyli miesięcznie daje mi to - te 6 centów?

**Reply by kamil-mrzyglod** [reply]
W Twoim wypadku w przybliżeniu zapewne tak będzie. Natomiast Logic Apps ze swoim pricing są nieco bardziej tricky - w ramach jednej appki możesz mieć kilka akcji do wykonania więc ostateczny rachunek za pojedyncze wykonanie może być wyższy niż "ilość wywołań * koszt jednej akcji".

**Reply by dzejzipl** [reply]
Tutaj ma być akurat jedna akcja po wykryciu nowego posta. Ale ok, już zrozumiałem. 
Po prostu profilaktycznie ustawię jakiś budżet na to.
Dzięki Kamil, zamykam! :)
